src/libvirt-xml-parse.js

```diff
@@ -402,6 +402,41 @@ export function parseDumpxmlForConsoles(devicesElem) {
     return displays;
 }
 
+export function parseDumpxmlForCapabilities(capabilitiesXML) {
+    const capabilitiesElem = getElem(capabilitiesXML);
+    const capabilities = { guests: [] };
+
+    if (capabilitiesElem) {
+        const guestElems = capabilitiesElem.getElementsByTagName('guest');
+        for (let i = 0; i < guestElems.length; i++) {
+            const guestElem = guestElems[i];
+
+            const osTypeElem = getSingleOptionalElem(guestElem, 'os_type');
+            const archElem = getSingleOptionalElem(guestElem, 'arch');
+
+            const guestCapabilities = { // see https://libvirt.org/formatdomain.html#elementsDisks
```

Comment:
Should that be `// see https://libvirt.org/formatcaps.html`?

Reply:
Err yes, https://libvirt.org/formatcaps.html#guest-capabilities -- thanks for spotting! Will push a fix once you are done with the review (in case you find other things)

Reply:
Fixed.